src/History.tsx

```diff
@@ -120,3 +85,70 @@ export class History extends React.Component<Props, {}> {
         );
     }
 }
+
+interface MessageProps {
+    activity: Activity;
+    autoscroll: () => void;
+    fromMe: boolean;
+    onClick?: React.MouseEventHandler<HTMLDivElement>;
+    selected: boolean;
+    showTimestamp: boolean;
+    store: ChatStore;
+    strings: Strings;
```

Comment:
I was trying to avoid a call to getState() for every message being rendered. Seemed repeatedly expensive.